dcc-portal-ui/Gruntfile.js

```diff
@@ -483,16 +483,33 @@ module.exports = function (grunt) {
           ignorePath: 'app'
         },
         files: [{
-          expand: true,
-          cwd: '<%= yeoman.app %>/develop/scripts',
-          src: ['*.js']
-        }]
+            expand: true,
+            cwd: '<%= yeoman.app %>/develop/scripts',
+            src: ['*.js']
+          },
+          {
+            expand: true,
+            cwd: '<%= yeoman.app %>/branding/scripts',
+            src: ['*.js']
+          }
+        ]
       }
     }
   });
 
   grunt.registerTask('bower-install', ['bower-install-simple']);
 
+  // A very basic task to read from process.env
+  grunt.registerTask('branding_customize', 'Override UI images and strings. see app/branding/scripts/config.js', function() {
+    if (process.env.ICGC_BRANDING) {
+      var _fileName = grunt.template.process('<%= yeoman.app %>/branding/scripts/custom.js');
+      grunt.file.write(_fileName, '$ICGC_BRANDING = ' + process.env.ICGC_BRANDING);
```

Comment:
@k1643 OK, the devops build would write 
`/dcc-portal-ui/app/branding/scripts/custom.js` and *not* set the branding variable.  
The injection config at https://github.com/ohsu-computational-biology/dcc-portal/blob/fa151a5edbc0503ae0ac0fe1c24a8d032ceaa9b7/dcc-portal-ui/Gruntfile.js#L490-L494 would pick that up.

Let me know if this is OK, and I will deprecate the variable